src/metabase/sync/sync_metadata/tables.clj

```diff
@@ -189,32 +209,32 @@
   ([database :- i/DatabaseInstance db-metadata]
    ;; determine what's changed between what info we have and what's in the DB
    (let [db-tables               (table-set db-metadata)
-         our-metadata            (our-metadata database)
-         strip-desc              (fn [metadata]
-                                   (set (map #(dissoc % :description) metadata)))
+         name+schema->db-table   (m/index-by (juxt :name :schema) db-tables)
+         our-metadata            (db->our-metadata database)
+         keep-table-id-info      (fn [metadata]
+                                   (set (map #(select-keys % [:schema :name]) metadata)))
          [new-tables old-tables] (data/diff
-                                  (strip-desc db-tables)
-                                  (strip-desc our-metadata))
-         [changed-tables]        (data/diff db-tables our-metadata)]
+                                  (keep-table-id-info db-tables)
+                                  (keep-table-id-info our-metadata))]
      ;; update database metadata from database
      (when (some? (:version db-metadata))
        (sync-util/with-error-handling (format "Error creating/reactivating tables for %s"
                                               (sync-util/name-for-logging database))
          (update-database-metadata! database db-metadata)))
      ;; create new tables as needed or mark them as active again
      (when (seq new-tables)
-       (sync-util/with-error-handling (format "Error creating/reactivating tables for %s"
-                                              (sync-util/name-for-logging database))
-         (create-or-reactivate-tables! database new-tables)))
+       (let [new-tables-info (set (map #(get name+schema->db-table [(:name %) (:schema %)]) new-tables))]
```

Comment:
previously, when creating a table, we didn't include a table description; we relied on the update step to do that. that's necessary, so I've changed it so that the description is included in the creation step.

Reply:
we can't fetch the value from the diff because the diff only contains `schema + name`, whereas `create-or-reactivate-tables` need to get the full info from `db-tables`.

Reply:
Yes, that's why I passed the diff output through `(->> <diff> (map schema+name->table))`

Sorry for the poor formatting I was taking shortcuts to write this in the github UI.

Here's the correct formatting:
```
[new-tables old-tables] (->> (data/diff (set (map schema+name db-tables)) (set (map schema+name our-metadata)))
                             (map schema+name->table))
```
Of course you could do this in two steps. My main point is that `schema+name` can be extracted out into a function and used to do the diff and getting the full info.

Reply:
did that here https://github.com/metabase/metabase/pull/36677/commits/ce46551884f37acbd888e40e00d1d377d738df2b#diff-8a4b5a6ee44180ba51e170fe36de9d5f3f889cd95b1ee56d19c72b6aac130ae5R215-R222